packages/compass-indexes/src/components/indexes-table/indexes-table.tsx

```diff
@@ -216,6 +205,12 @@ export function IndexesTable<Column extends string>({
                   </Cell>
                 )}
                 {info.details && (
+                  // TODO(COMPASS-7046): leafygreen animation for the nested row
```

Comment:
Existing issue, just leaving a reference so that we can double-check if we can fix this after leafygreen update